src/commands/shuffle-voice-channel-names/index.ts

```diff
@@ -25,9 +27,11 @@ class ShuffleVoiceChannelNamesCommand extends Command {
     await interaction.deferReply({ ephemeral: true });
 
     const guild = interaction.guild!;
+    const member = interaction.member as GuildMember;
```

Comment:
This is unnecessary if I am not tagging the user. `interaction.user` will work instead

Reply:
I'd say use the smaller type just for _more correctness_